index.hbs

```diff
@@ -1,17 +1,14 @@
 {{!< default}}
-<ol class="posts">
+
+<h1 style="text-align: center;">Writtings.</h1>
+<ol id="posts-list">
   {{#foreach posts}}
   <li>
-    <a href="{{url}}" title="link to {{{title}}}">
-      <h2>{{{title}}}</h2>
-      <p class="post description">{{excerpt}}&hellip;</p>
```

Comment:
What's your point for do it?

Most of the time description is not useful and looks like generic text IMHO

On Sunday, December 6, 2015, Lou Marvin Caraig <notifications@github.com>
wrote:

> In index.hbs
> <https://github.com/Kikobeats/uno-zen/pull/135#discussion_r46774491>:
>
> >    {{#foreach posts}}
> >    <li>
> > -    <a href="{{url}}" title="link to {{{title}}}">
> > -      <h2>{{{title}}}</h2>
> > -      <p class="post description">{{excerpt}}&hellip;</p>
>
> is it possible in some way to reenable the posts description?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Kikobeats/uno-zen/pull/135/files#r46774491>.
>